src/Routing/Filter/ThrottleFilter.php

```diff
@@ -22,28 +49,18 @@ public function __construct($config = [])
         $config += [
             'priority' => 1,
             'message' => 'Rate limit exceeded',
-            'interval' => '1 minute',
+            'interval' => '+1 minute',
             'rate' => 10,
             'identifier' => function (Request $request) {
                 return $request->clientIp();
             },
+            'headers' => true,
+            'rateHeader' => 'X-Throttle-Limit',
+            'remainingHeader' => 'X-Throttle-Remaining',
+            'resetHeader' => 'X-Throttle-Reset'
```

Comment:
I had that originally and it does indeed feel better. The reason I changed it to this was because it broke when not all headers were configured but that should be catchable. Will revert.